### ISSUE TITLE
Consider using UTF-8 in JavaScript

### ISSUE THREAD

**Issue by ylemkimon** [KaTeX/KaTeX#1598]
Currently, source code and minified code contains only ASCII:
https://github.com/Khan/KaTeX/blob/51bf3fa6f51f566fdf6c6f1ee900e7f986014a80/webpack.common.js#L112-L116

By using UTF-8 we can save a few kilobytes. This needs setting charset by one of following ways:
* charset in `Content-Type` HTTP response header
* `<meta charset>` in the `head`
* `charset` attribute in the `script` tag

CDNs already serve with UTF-8 `Content-Type` header.

However, for files served from servers without charset set, setting charset is needed, which may be a breaking change.

**Comment by kevinbarabash** [comment]
Interesting.  If we changed that to `false` will it also converted unicode that's escaped to unicode chars?  I guess as long as people are using this from CDNs they'll be alright but some people may be hosting the files themselves.  I think it should be okay as long as we making it a breaking change when we the release this change.

**Comment by ylemkimon** [comment]
@kevinbarabash Yes, UglifyJS will convert escaped unicode to unicode char and encode them in UTF-8.

**Comment by edemaine** [comment]
This would definitely break several installations (e.g. mine).  But it's an easy fix, adding `charset` to the `<script>` tag.  I guess the question is whether the space savings is worth it...  (It's a shame that utf-8 isn't the default charset.)
